include/dem/grid_motion.h

```diff
@@ -54,8 +55,9 @@ class GridMotion
    * @param dem_parameters DEM parameters defined using the parameter handler
    * @param dem_time_step DEM time-step
    */
-  GridMotion(const DEMSolverParameters<dim> &dem_parameters,
-             const double &                  dem_time_step);
+  GridMotion(const DEMSolverParameters<dim> &     dem_parameters,
+             const double &                       dem_time_step,
+             std::shared_ptr<PWContactForce<dim>> pw_contact_force_object = {});
```

Comment:
Does the GridMotion class need to have access to the pw_contact_force_object? or can we just send the forces from the pw_contact_force to the GridMotion?

Reply:
Yes, i think it's true !